website/content/docs/nia/terraform-modules.mdx

```diff
@@ -36,6 +36,8 @@ First, create a directory to organize Terraform configuration files that make up
 
 The `main.tf` is the entry point of the module and this is where you can begin authoring your module. It can contain multiple Terraform resources related to an automation task that uses Consul service discovery information, particularly the required [`services` input variable](#services-variable). The code example below shows a resource using the `services` variable. When this example is used in automation with Consul-Terraform-Sync, the content of the local file would dynamically update as Consul service discovery information changes.
 
+<CodeBlockConfig filename="main.tf">
```

Comment:
TIL! this is a really nice QOL addition. Thanks @blake 
![image](https://user-images.githubusercontent.com/6362111/129098726-92502ba0-4f9e-4eae-bc15-bbb72df79604.png)